source/includes/language-compatibility-table-java-rs.rst

```diff
@@ -9,6 +9,18 @@
      - Java 7
      - Java 6
 
+   * - 4.5
+     - |checkmark|
+     - |checkmark|
+     -
+     -
+
+   * - 4.4
+     - |checkmark|
+     - |checkmark|
+     -
+     -
+
```

Comment:
Any reason not to port this file over to using '✓' as the other files have been moved to that?

Reply:
No good reason, but can address that in another PR to consolidate all of them, especially if we pursue the Google sheet-based solution